src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -213,34 +226,89 @@ const useGroupInfo = (channelId = null) => {
     }
   };
 
-  const updateMemberName = (members = []) => {
-    if (!channel.data.isEditName && members.length > 0) {
-      members = members.map((member) => member.user.name).join(',');
-      setUsername(members);
-      if (members.length > 1) {
-        channel?.update({
-          name: members
-        });
-      }
+  const onRemoveUser = async () => {
+    setOpenModal(false);
+    const responseChannelData = await removeMemberGroup({
+      channelId,
+      targetUserId: selectedUser?.userId
+    });
+
+    try {
+      const {channelName} = getChannelListInfo(
+        responseChannelData.data,
+        signedProfileId,
+        anonProfileId
+      );
+      const channelList = await ChannelListSchema.getSchemaById(localDb, channelId);
+      channelList.name = channelName;
+      await channelList.save(localDb);
+
+      setSelectedChannel(channelList);
+
+      await UserSchema.deleteByUserId(localDb, selectedUser?.userId, channelId);
+    } catch (e) {
+      console.log('error on memberSchema');
+      console.log(JSON.stringify(e));
+      console.log(e);
     }
+
+    refresh('channelList');
+    refresh('chat');
+    refresh('channelInfo');
+
+    setTimeout(() => {
+      navigation.navigate('SignedChatScreen');
+    }, 500);
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onRemoveUser` function has been refactored to use the `removeMemberGroup` service and update the local database accordingly. The error handling could be improved by providing more specific error messages instead of just logging 'error on memberSchema'. Also, consider using a logger utility instead of console.log for better logging management.
```diff
- console.log('error on memberSchema');
+ console.error('Error while updating member schema in onRemoveUser:', e);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->